gammapy/spectrum/flux_point.py

```diff
@@ -18,12 +19,379 @@
+
+        # set flux points plotting defaults
+        kwargs.setdefault('marker', 'None')
+        kwargs.setdefault('ls', 'None')
+
+        ebar = ax.errorbar(x[~is_ul].value, y[~is_ul].value, yerr=y_err,
+                           xerr=x_err, **kwargs)
+
+        if is_ul.any():
+            if x_err_all:
+                x_errn, x_errp = x_err_all
+                x_err = (x_errn[is_ul].to(energy_unit).value,
+                         x_errp[is_ul].to(energy_unit).value)
+
+            y_ul = self.table[sed_type + '_ul'].quantity
+            y_ul = (y_ul * np.power(x, energy_power)).to(y_unit)
+
+            # set ul plotting defaults
+            ul_kwargs = {'marker': 'v',
+                         'label': None}
+
+            kwargs.setdefault('ms', 10)
+            kwargs.setdefault('mec', 'None')
+            kwargs.setdefault('c', ebar[0].get_color())
+            kwargs.update(ul_kwargs)
+
+            ax.errorbar(x[is_ul].value, y_ul[is_ul].value, xerr=x_err, **kwargs)
+
+        ax.set_xscale('log', nonposx='clip')
+        ax.set_yscale('log', nonposy='clip')
+        return ax
+
+    def _plot_get_x_err(self, sed_type):
+        try:
+            e_min = self.table['e_min'].quantity
+            e_max = self.table['e_max'].quantity
+            e_ref = np.sqrt(e_min * e_max)
+            x_err = ((e_ref - e_min), (e_max - e_ref))
+        except KeyError:
+            x_err = None
+        return x_err
+
+    def _plot_get_y_err(self, sed_type):
+        try:
+            # assymmetric error
+            y_errn = self.table[sed_type + '_errn'].quantity
+            y_errp = self.table[sed_type + '_errp'].quantity
+            y_err = (y_errn, y_errp)
+        except KeyError:
+            try:
+                # symmetric error
+                y_err = self.table[sed_type + '_err'].quantity
+                y_err = (y_err, y_err)
+            except KeyError:
+                # no error at all
+                y_err = None
+        return y_err
+
+    @property
+    def _is_ul(self):
+        try:
+            return self.table['is_ul'].data.astype('bool')
+        except KeyError:
+            return np.isnan(self.table[self.sed_type])
+
+    def show(self, figsize=(8, 5), **kwargs):
+        """
+        Show flux points.
+
+        Parameters
+        ----------
+        figsize : tuple
+            Figure size
+        kwargs : dict
+            Keyword arguments passed to `FluxPoints.plot()`.
+
+        Returns
+        -------
+        ax : `~matplotlib.axes.Axes`
+            Plotting axes object.
+        """
+        import matplotlib.pyplot as plt
+
+        fig = plt.figure(figsize=figsize)
+        ax = fig.add_subplot(111)
+        self.plot(ax=ax, **kwargs)
+        return ax
+
+    def __str__(self):
+        """
+        String representation of the flux points class.
+        """
+        info = ''
+        info += "Flux points of type '{}'".format(self.sed_type)
+        return info
+
+    def info(self):
```

Comment:
Suggest to remove `info` and just keep `__str__`.
This is what we've done for most classes, at least for now.